<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceActivity</class>
 <widget class="QWidget" name="FaceActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <widget class="EyesWidget" name="widgetEyes" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="RealtimeValuesWidget" name="widgetRealtimeValues" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="LogWidget" name="logScroll">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="ConnectionStatusWidget" name="widgetConnectionStatus" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogWidget</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">log/LogWidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>EyesWidget</class>
   <extends>QWidget</extends>
   <header location="global">expression/EyesWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RealtimeValuesWidget</class>
   <extends>QWidget</extends>
   <header location="global">agent/RealtimeValuesWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConnectionStatusWidget</class>
   <extends>QWidget</extends>
   <header location="global">connection/ConnectionStatusWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>splitter</sender>
   <signal>splitterMoved(int,int)</signal>
   <receiver>FaceActivity</receiver>
   <slot>onSplitterMoved(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>865</x>
     <y>658</y>
    </hint>
    <hint type="destinationlabel">
     <x>865</x>
     <y>675</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSplitterMoved(int,int)</slot>
  <slot>panicChanged(bool)</slot>
  <slot>randomizeColor()</slot>
 </slots>
</ui>
